sdl2promela/sdl/model.py

```diff
@@ -53,6 +129,55 @@ class Task(Action):
     pass
 
 
+class AssignmentTask(Task):
+    """Assignment action."""
+
+    assignment: BinaryExpression
+    """Assignment expression."""
+
+
+class ForLoopRange:
+    """Base class for ranges that can be iterated over"""
+
+    pass
+
+
+class NumericForLoopRange(ForLoopRange):
+    """Numeric range that can be iterated over"""
+
+    start: Expression
+    """Start of the range, inclusive."""
+
+    stop: Expression
+    """End of the range, exclusive."""
+
+    step: Expression
+    """Iteration step, defaults to 1."""
+
+    def __init__(self):
+        self.start = None
+        self.stop = None
+        self.step = Constant("1")
```

Comment:
ou, so integer is just string

Reply:
String is good enough for now, the approach may be changed for complex types.